yin_yang/NotificationHandler.py

```diff
@@ -1,33 +1,31 @@
 from logging import Handler
 
-from PySide6.QtDBus import QDBusMessage, QDBusConnection
+from PySide6.QtDBus import QDBusConnection, QDBusMessage
 
 
 def create_dbus_message(title: str, body: str):
     message = QDBusMessage.createMethodCall(
-        'org.freedesktop.portal.Desktop',
```

Comment:
There are a ton of these changes in quotations marks and line separators, which makes it really hard to see if anything changed at all. Is there a specific reason why you changed them, or just to satisfy the linter?

I personally prefer single quotation (`'`), and I think we should stick to them just to keep changes to a minimum.

Reply:
I saw that since pylint and flake8 are being used, I set up my environment to use these as well. I'll update the pylintrc file to use single quotes and fix these changes.